crates/wasm-wave/src/value/wit.rs

```diff
@@ -155,10 +155,7 @@ mod tests {
 
     #[test]
     fn resolve_wit_type_smoke_test() {
-        let UnresolvedPackageGroup {
-            mut packages,
-            source_map,
-        } = UnresolvedPackageGroup::parse(
+        let UnresolvedPackageGroup { mut main, .. } = UnresolvedPackageGroup::parse(
```

Comment:
Mind updating these tests to using `resolve.push_str` instead of using `UnresolvedPackageGroup`?